src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx

```diff
@@ -12,6 +12,76 @@ import {Client} from 'app/api';
 import {update} from 'app/actionCreators/projects';
 import overflowEllipsis from 'app/styles/overflowEllipsis';
 
+class PlatformList extends React.Component {
+  static propTypes = {
+    platforms: PropTypes.arrayOf(PropTypes.string),
```

Comment:
not required, would be v nice if we had a list of valid platform strings in `SentryTypes`

Reply:
Yeah, although I'm not sure what they are and if they are subject to change

Reply:
Yeah the reason why it'd be nice to have is that it's difficult to find the list.

Looks like this is the current set of platforms: https://sentry.io/settings/sentry/javascript/install/